tests/cover/test_cache_cls.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+"Test new caching classes"
+
+#PyFPDF-cover-test:res=font/DejaVuSansCondensed.ttf
+#PyFPDF-cover-test:res=font/DejaVuSans.ttf
+
+import common
+import fpdf
+
+import os, shutil, time
+import gzip
+
+def testfile(f1, f2, cache):
+    # create pdf
+    pdf = fpdf.FPDF()
+    pdf.set_cache(cache)
+    if f1:
+        pdf.add_font('DejaVuSansCondensed', '', f1, uni = True)
+    if f2:
+        pdf.add_font('DejaVuSans', '', f2, uni = True)
+        pdf.set_font('DejaVuSans', "", 10)
+    return pdf
+
+def trashfile(fn):
+    f = open(fn, "w")
+    f.write("1234567890")
+    f.close()
+
+try:
+    from hashlib import md5
```

Comment:
hashlib was apparently added in Python 2.5, so this juggling of imports may not be necessary